main/iostream.c

```diff
@@ -597,6 +608,7 @@ static void iostream_dtor(void *cookie)
 	 * put this in a conditional to avoid compiler warnings. */
 	struct ast_iostream *stream = cookie;
 #endif
+	ast_free(stream->sni_hostname);
```

Comment:
This won't work in non-dev mode as stream is not present.

Reply:
done